environments/testing.toml

```diff
@@ -1,5 +1,5 @@
 [database]
-    name     = "vulcanize_private"
+    name     = "vulcanize_testing"
```

Comment:
Is this config used when we run the tests? If so, I'd be tempted to remove the `ipcPath` and verify that there's a warning to add one if you try to run the tests with it empty.

Also, I think such a warning does currently exist that points at `infura.toml` - probably worth making sure that's up to date if we're using this one now

Reply:
Yees, warn for that! My tests fail regularly from me not setting that field. 

Would also be cool to prune some configs we're not using, since I'm always wondering which `toml` is going to be used for tests :woman_shrugging:

Reply:
Added a warning when there is no `ipcPath` set in the file or via env variable. Using `vulcanize_testing` and removing `vulcanize_private` was an attempt to prune configs and make it consistent with mcdTransformers.